bin/build

```
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+function usage() {
+    echo -e "Usage: bin/build [TAG]
+
+OPTIONS:
+  -h, --help   print this message
+
+ARGUMENTS:
+  TAG        image tag in the format name:tag
+"
+}
+
+
+# Build moodlenet-backend image
+#
+# Usage: build TAG
+function build() {
+    tag="${1}"
+    version=$(cat docker/images/backend/UPSTREAM_VERSION)
+    docker_user="$(id -u):$(id -g)"
+
+    echo -e "${COLOR_INFO}Building the moodlenet-backend image with " \
+    "upstream version ${version}.${COLOR_RESET}"
```

This dot may be confusing after a version number?

```suggestion
    "upstream version ${version}${COLOR_RESET}"
```

---

Good catch!